packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -597,6 +597,9 @@ class VirtualizedList extends React.PureComponent<Props, State> {
         ref: this,
       });
     }
+    if (this.props.inverted) {
+      this._invertScrolling(true);
```

Comment:
My thought is that this should probably be done in the platform ScrollView, rather than this component. I think this list passes all its props to ScrollView, even though ScrollView doesn't technical support that prop